### PR TITLE
encourage all co-instructors to attend

### PR DESCRIPTION
**Pull request by raynamharris**
I started thinking more about potential ways to module the number of attendees and workshops at our discussion session. One potentially easy way to achieve our ideal numbers is to be more clear about inviting co-instructors to attend the same discussion session. 

What do you think about adding a sentence like this? How could this sentence be improved?

@ChristinaLK @tracykteal @maneesha @smcclatchy

### CODE REVIEW

File: subcommittees/mentoring/files/pre_workshop_email.txt [hunk 1]
```diff
@@ -17,8 +17,12 @@ for the most convenient time on this etherpad:
 On the day of the meeting, connection information will be posted on the
 same etherpad.
 
+We encourage all workshop co-insturctors to attend the same discussion
```
Comment (ChristinaLK) [inline subcommittees/mentoring/files/pre_workshop_email.txt]: typo: co-insturctors -> co-instructors

**Comment by ChristinaLK** [comment]
I'm +1 to adding it to both emails - thinking through the wording still.

**Comment by maneesha** [comment]
What about:

> We encourage workshop co-instructors to attend the same session to minimize redundant conversations and keep these discussions flowing smoothly.  

A bit more transparent but also a bit more blunt :).

**Comment by raynamharris** [comment]
Thanks @maneesha! I like this wording a lot.

**Pull request merged**